
Including the old termcap.h and the term.h results in mixed prototypes ...

  - Rene Rebe <rene@exactcode.de>

diff -u psmisc-21.5/src/pstree.c psmisc-21.5-fixed/src/pstree.c
--- psmisc-21.5/src/pstree.c	2004-04-24 14:44:27.000000000 +0200
+++ psmisc-21.5-fixed/src/pstree.c	2004-12-21 19:04:03.265558192 +0100
@@ -18,7 +18,7 @@
 #include <curses.h>
 #include <term.h>
 #include <termios.h>
-#include <termcap.h>
+/*#include <termcap.h>*/
 #include <langinfo.h>
 #include <sys/types.h>
 #include <sys/stat.h>
